span.go

```diff
@@ -141,6 +141,11 @@ func (s *Span) LogEvent(event string) {
 	s.Log(opentracing.LogData{Event: event})
 }
 
+// Logs returns the Span logs
+func (s *Span) Logs() []opentracing.LogRecord {
+	return s.logs
```

Comment:
this needs to be wrapped with RLock(). If we do expose this function, the returned logs should be immutable.